web/client/translations/data.it-IT.json

```
@@ -1982,6 +1982,9 @@
                 "brightTextThemeLabel": "Testo chiaro",
                 "darkThemeLabel": "Scuro",
                 "darkTextThemeLabel": "Testo scuro",
+                "defaultThemeLabel": "Predefinito",
+                "customizeThemeLabel": "Personaliza",
+                "customizeThemeRemoveLabel": "Rimuovere tema personalizzato",
```

I think that if we use "Personalizza", "mostra" "usa" we should use "Rimuovi" instead of "Rimuovere"
```suggestion
                "customizeThemeRemoveLabel": "Rimuovi il tema personalizzato",
```